homeassistant/components/enigma2/quality_scale.yaml

```diff
@@ -0,0 +1,74 @@
+rules:
+  # Bronze
+  action-setup:
+    status: exempt
+    comment: Only entity services
+  appropriate-polling: done
+  brands: done
+  common-modules: done
+  config-flow-test-coverage: done
+  config-flow: done
+  dependency-transparency: done
+  docs-actions: done
+  docs-high-level-description: done
+  docs-installation-instructions: done
+  docs-removal-instructions: done
+  entity-event-setup:
+    status: exempt
+    comment: The integration registers no events
+  entity-unique-id: done
+  has-entity-name: done
+  runtime-data: done
+  test-before-configure: done
+  test-before-setup: done
+  unique-config-entry: done
```

Comment:
How does this work?
```py

            unique_id = about["info"]["ifaces"][0]["mac"] or self.unique_id
            await self.async_set_unique_id(unique_id)
            self._abort_if_unique_id_configured()
```

What is `self.unique_id`? If this is a new flow, the unique_id is just nothing right?